gamemodes/terrortown/entities/entities/ttt_credit_adjust.lua

```diff
@@ -20,7 +20,7 @@ function ENT:KeyValue(key, value)
 		self.Credits = tonumber(value) or 0
 
 		if not tonumber(value) then
-			ErrorNoHalt(tostring(self) .. " has bad 'credits' setting.\n")
+			ErrorNoHaltWithStack(tostring(self) .. " has bad 'credits' setting.\n")
```

Comment:
Could you explain here the helpfulness of this? Because this is also called from a map entity, meaning that the stacktrace will probably not point to any error that actually exists.

I'm not entirely sure though, that's just my impression. Feel free to say I'm wrong here